concent_api/core/payments/backends/sci_backend.py

```diff
@@ -51,10 +55,46 @@ def get_list_of_payments(
             from_block      = last_block_before_payment,
             to_block        = payment_interface.get_block_number(),  # pylint: disable=no-member
         )
-
     return payments_list
 
 
+def validate_that_last_closure_time_is_older_than_last_payment(
+    requestor_eth_address: str,
+    provider_eth_address: str,
+    youngest_payment_ts: Optional[int],
+) -> None:
+    if youngest_payment_ts is None:
+        return
```

Comment:
After fixing @dybi comment above this should be removed, but either way we `return None` explicitly.

Reply:
OK, thanks